prettyPrinters/rust_types.py

```diff
@@ -34,21 +34,21 @@ class RustType(object):
 
 
 # Should be synchronized with `RsDebugProcessConfigurationHelper.RUST_STD_TYPES`
-STD_STRING_REGEX = re.compile(r"^(alloc::(\w+::)+)String$")
+STD_STRING_REGEX = re.compile(r"^(alloc::([a-z_]+::)+)String$")
```

Comment:
Because our LLDB for MSVC does not support `\w` metacharacter. There is the corresponding `[[:alnum:]]` metacharacter supported by our LLDB, but it does not allow `_`. However, `[a-z_]` seems to work fine.

Reply:
Let's add the corresponding comment somewhere to make new pretty printer creation easier

Reply:
There is a note in the commit message body, but I can add a comment to the code

Reply:
I think explicit comment in code is better because if someone wants to create a new pretty printer, they hardly read your commit message